testdata/src/go.uber.org/receivers/inference/receivers-with-inference.go

```diff
@@ -111,3 +111,51 @@ func testAffiliation() {
 	// FP since affiliations are not tracked for nilable receivers
 	newI2().foo() //want "result 0 of `newI2.*`"
 }
+
```

Comment:
Can we also add cases where the receiver is the blank identifier (`func (_ *A) foo()`)?

Reply:
Good idea. Added.